pkg/security/probe/process_killer.go

```diff
@@ -30,14 +31,32 @@ type ProcessKiller struct {
 	sync.Mutex
 
 	pendingReports   []*KillActionReport
-	binariesExcluded []string
+	binariesExcluded []*eval.Glob
 }
 
 // NewProcessKiller returns a new ProcessKiller
-func NewProcessKiller(cfg *config.Config) *ProcessKiller {
-	return &ProcessKiller{
-		binariesExcluded: append(binariesExcluded, cfg.RuntimeSecurity.EnforcementBinaryExcluded...),
+func NewProcessKiller(cfg *config.Config) (*ProcessKiller, error) {
+	var p ProcessKiller
+
+	for _, str := range binariesExcluded {
+		glob, err := eval.NewGlob(str, false, false)
+		if err != nil {
+			return nil, err
+		}
+
+		p.binariesExcluded = append(p.binariesExcluded, glob)
+	}
+
+	for _, str := range cfg.RuntimeSecurity.EnforcementBinaryExcluded {
+		glob, err := eval.NewGlob(str, false, false)
+		if err != nil {
+			return nil, err
+		}
+
+		p.binariesExcluded = append(p.binariesExcluded, glob)
 	}
```

Comment:
Nit: Maybe we could range on an append() of cfg.RuntimeSecurity.EnforcementBinaryExcluded + binariesExcluded to avoid code duplication ?